test/bdd/fixtures/docker-compose.yml

```diff
@@ -20,6 +20,7 @@ services:
       - KMSSECRETS_DATABASE_TYPE=mongodb
       - KMSSECRETS_DATABASE_URL=mongodb://mongodb.example.com:27017
       - KMSSECRETS_DATABASE_PREFIX=issuer_
+      - VC_SECRET_LOCK_KEY_PATH=/etc/tls/secret-lock.key
```

Comment:
How is key rotation handled for KMS secret lock ?

Reply:
Not handled now, the same problem as with key rotation in KMS. We can implement rotation, but I think it better to create a separate issue, because it is not trivial

Reply:
Yes, let's create an issue - this is an important topic.

Reply:
https://github.com/trustbloc/vcs/issues/783